### PR TITLE
Sylius 1.11 support

### PR DESCRIPTION
**Pull request by marekrzytki**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -16,13 +16,17 @@ jobs:
             fail-fast: false
```
Comment (jakubtobiasz) [inline .github/workflows/build.yml:16]: Can you add Sylius version to run name like here https://github.com/BitBagCommerce/SyliusWishlistPlugin/blob/master/.github/workflows/build.yml?

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -16,13 +16,17 @@ jobs:
...
                 exclude:
                     - # Segmentation fault while clearing cache
                         php: 8.0
                         mysql: 5.7
+                    -
+                        sylius: ~1.11.0
+                        php: 7.4
 
         env:
             APP_ENV: test
```
Comment (jakubtobiasz) [inline .github/workflows/build.yml:35]: There's no restricting Sylius version. We need to add this. After restricting probably exluding Sylius 1.9 and PHP 8 is going to be needed. Ref workflow https://github.com/BitBagCommerce/SyliusWishlistPlugin/blob/master/.github/workflows/build.yml

File: tests/Application/.env [hunk 3]
```diff
@@ -21,6 +21,14 @@ JWT_PUBLIC_KEY=%kernel.project_dir%/config/jwt/public.pem
 JWT_PASSPHRASE=acme_plugin_development
 ###< lexik/jwt-authentication-bundle ###
 
+###> symfony/messenger ###
+# Choose one of the transports below
+# MESSENGER_TRANSPORT_DSN=amqp://guest:guest@localhost:5672/%2f/messages
+#MESSENGER_TRANSPORT_DSN=doctrine://default
```
Comment (jakubtobiasz) [inline tests/Application/.env]:
```suggestion
# MESSENGER_TRANSPORT_DSN=doctrine://default
```

**Review by jakubtobiasz — CHANGES_REQUESTED** [review]
After adding a Sylius' version restricting there may appear additional things to fix :P.

File: .github/workflows/build.yml (hunk 1 above)
Comment (marekrzytki) [reply .github/workflows/build.yml:16]: Done

File: .github/workflows/build.yml (hunk 2 above)
Comment (marekrzytki) [reply .github/workflows/build.yml:35]: Done

**Review by senghe — APPROVED** [review]

**Pull request merged**